src/components/Incidents/IncidentList/index.js

```diff
@@ -57,6 +56,14 @@ function IncidentItem({ incident }) {
   const navigateToIncidentDetails = (id) => {
     navigate(`/incidents/${id}`);
   };
+
+  const getAvatar = (avatar) => {
```

Comment:
Can we use AvatarGroup for this: https://github.com/flanksource/flanksource-ui/pull/190/files#diff-e2b6e18b1e98619f24f1d861fef48fad9706a512765e189db37b131b7be153ac ?